src/IndexBuilder.php

```diff
@@ -22,9 +22,10 @@ public function __construct(Client $client, $configurationDirectory)
         $this->configurationDirectory = $configurationDirectory;
     }
 
-    public function createIndex($indexName): Index
+    public function createIndex($indexName, $fileName = null): Index
     {
-        $mappingFilePath = $this->configurationDirectory.DIRECTORY_SEPARATOR.$indexName.'_mapping.yaml';
+        $fileName = $fileName ?? $indexName;
+        $mappingFilePath = $this->configurationDirectory.DIRECTORY_SEPARATOR.$fileName.'_mapping.yaml';
```

Comment:
```suggestion
        $fileName = $fileName ?? ($indexName.'_mapping.yaml');
        $mappingFilePath = $this->configurationDirectory.DIRECTORY_SEPARATOR.$fileName;
```
Why not this version ? In order to allow more possible fileName and remove that `_mapping.yaml` part.

Reply:
Yes indeed, thanks :)